zerver/lib/test_helpers.py

```diff
@@ -421,6 +421,7 @@ def find_pattern(pattern, prefixes):
 
         if full_suite:
             print('INFO: URL coverage report is in %s' % (fn,))
+            print('INFO: View report at http://127.0.0.1:9991/coverage/index.html')
```

Comment:
This section of code in `test_helpers.py` is for the "URL coverage" report, which is separate from the "test coverage" report, and unrelated to that URL.